docs/src/components/PageHeader.js

```diff
@@ -47,8 +47,8 @@ export default function ComponentHeader({
         </Heading>
         {showSourceLink && (
           <Text color="gray">
-            <Link href={githubUrl(fileName || name)} inline>
-              Source
+            <Link href={githubUrl(fileName || name)} inline target="blank">
+              View source on Github
```

Comment:
Should be GitHub

Reply:
I'm just wondering if spacing out a little bit more the link could mprove the visibility. I'd like to hear @ashleyseto opinion, but I think many people skip this line due to being shadowed by the big header...

Reply:
I'm not sure exactly how line height would translate but putting in 4px of spacing in between the title and link seems to balance it out better.